components/compliance-service/dao/pgdb/migration/sql/42_create_data_lifecycle.up.sql

```diff
@@ -0,0 +1,8 @@
+
+CREATE TABLE IF NOT EXISTS compliance_lifecycle (
+  id BIGSERIAL PRIMARY KEY,
+  policy_name TEXT NOT NULL,
+  older_than_days INTEGER NOT NULL
```

Comment:
Do we want to store the updated time as well? Rather than older_than_days , can we rename to policy_value?